testng-core-api/src/main/java/org/testng/internal/RuntimeBehavior.java

```diff
@@ -16,13 +19,20 @@ public final class RuntimeBehavior {
   public static final String TESTNG_DEFAULT_VERBOSE = "testng.default.verbose";
   public static final String IGNORE_CALLBACK_INVOCATION_SKIPS = "testng.ignore.callback.skip";
   public static final String SYMMETRIC_LISTENER_EXECUTION = "testng.listener.execution.symmetric";
+  public static final String IDE_PACKAGE_NAMES = "testng.ide.listeners.package.name";
```

Comment:
Not necessary for IDE only. Just rename it into a more generic name.

Reply:
Fixed this. Can you please check now ?